server/__tests__/createExpressHandler.test.ts

```diff
@@ -71,4 +77,63 @@ describe('the createExpressHandler function', () => {
   it('should correctly initialize a Twilio client', () => {
     expect(Twilio).toHaveBeenCalledWith('mockApiKeySid', 'mockApiKeySecret', { accountSid: 'mockAccountSid' });
   });
+
+  describe('the twilioClient.tokens.create() function call', () => {
+    let mockProcessExit: any;
+    let mockConsoleError: any;
+
+    beforeEach(() => {
+      mockConsoleError = jest.spyOn(console, 'error').mockImplementation();
```

Comment:
Nice! I've been looking for a way to get console.error to keep quiet. There are some "errors" logged by the react video app tests that don't need to be.